cmd/ssh_usercert_gen/main.go

```diff
@@ -55,7 +56,7 @@ type AppConfigFile struct {
 type RuntimeState struct {
 	Config              AppConfigFile
 	SSHCARawFileContent []byte
-	Signer              *ssh.Signer
+	Signer              *interface{}
```

Comment:
Yuk. This is the moral equivalent of the void pointer. Can you define a real interface type with the methods that you need? Also, interface types are already pointers, so you don't need to have a pointer to an interface type.